CJBItemSpawner/i18n/th.json

```diff
@@ -0,0 +1,31 @@
+{
+    /*********
+    ** Sort button
+    *********/
+    "sort.by-name": "ชื่อ",
+    "sort.by-type": "ประเภท",
+    "sort.by-id": "ID",
```

Comment:
This should probably be translated too. (ID means the internal item identifier.)

Reply:
@Pathoschild ID or some simple English word and generally used, Thai people can understand better than to translate into Thai